taxi/admin.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        (
+            "Additional info",
+            {"fields": ("license_number",)}
+        ),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        (
+            "Personal info",
+            {
+                "fields": (
+                    "first_name",
+                    "last_name",
+                    "email",
+                )
+            },
+        ),
+        (
+            "Additional info",
+            {
+                "fields": (
+                    "license_number",
+                )
+            },
+        ),
+    )
```

Comment:
I don't think there is a need in such complex indentation, but if you better understand structure with that its ok)